pkg/util/util.go

```diff
@@ -327,3 +327,34 @@ func Intersection(a []string, b []string) ([]string, []string) {
 
 	return matches, misses
 }
+
+// ParseMetric parses display name and type of field from the raw name of the metric as defined in (sub)template.
+// Users can rename a metric with "=>" (e.g. some_long_metric_name => short).
+// Trailing "^" characters are ignored/cleaned as they have special meaning in some collectors.
+func ParseMetric(rawName string) (string, string, string) {
```

Comment:
could u add hyphon to underscore conversion in this method.

Reply:
added.